<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LP1-AP-51836313" name="RetryMultipleFailedTests" tests="8" failures="8" timestamp="21 May 2021 18:05:22 GMT" time="0.032" errors="0">
  <testcase name="test1" time="0.008" classname="com.retrylogic.MyTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.retrylogic.MyTest.test1(MyTest.java:11)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.001" classname="com.retrylogic.MyTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.retrylogic.MyTest.test1(MyTest.java:11)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.0" classname="com.retrylogic.MyTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.retrylogic.MyTest.test1(MyTest.java:11)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.0" classname="com.retrylogic.MyTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.retrylogic.MyTest.test1(MyTest.java:11)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test2" time="0.0" classname="com.retrylogic.MyTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.retrylogic.MyTest.test2(MyTest.java:17)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.001" classname="com.retrylogic.MyTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.retrylogic.MyTest.test2(MyTest.java:17)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.0" classname="com.retrylogic.MyTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.retrylogic.MyTest.test2(MyTest.java:17)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.0" classname="com.retrylogic.MyTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.retrylogic.MyTest.test2(MyTest.java:17)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- RetryMultipleFailedTests -->
